frontend/src/features/mapSideBar/tabs/InventoryTabs.tsx

```diff
@@ -10,17 +13,55 @@ export enum InventoryTabNames {
   property = 'property',
   title = 'title',
   value = 'value',
+  styles = 'styles',
 }
 /**
  * Tab wrapper, provides styling and nests form components within their corresponding tabs.
  * @param param0 object containing all react components for the corresponding tabs.
  */
 export const InventoryTabs: React.FunctionComponent<IInventoryTabsProps> = ({ PropertyForm }) => {
+  const longText =
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam porttitor nisl at elit vestibulum vestibulum. Nullam eget consectetur felis, id porta eros. Proin at massa rutrum, molestie lorem a, congue lorem.';
   return (
     <TabView defaultActiveKey={InventoryTabNames.property}>
       <Tab eventKey={InventoryTabNames.title} title="Title"></Tab>
       <Tab eventKey={InventoryTabNames.value} title="Value"></Tab>
       <Tab eventKey={InventoryTabNames.property} title="Property Details"></Tab>
+      <Tab eventKey={InventoryTabNames.styles} title="STYLES">
```

Comment:
I suppose we'll take this out sometime during the sprint? or left in by accident?